File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/ordprintdto/d/OrdPrintParamDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.ordprintdto.d
{
    /// <summary>
    /// OrdPrintParamDTO 的摘要说明。
    /// </summary>
    public class OrdPrintParamDTO : BaseDTO {

        public OrdPrintParamDTO() {
 
        }

        /// <summary>
        /// 长临标识
        /// </summary>
		public bool? Fg_long {
            get { return getAttrVal<FBoolean>("Fg_long",null); }
            set { setAttrVal<FBoolean>("Fg_long", value); }
        }

        /// <summary>
        /// 患者Id
        /// </summary>
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }

        /// <summary>
        /// 就诊Id
        /// </summary>
		public string Id_en {
            get { return getAttrVal<string>("Id_en",null); }
            set { setAttrVal<string>("Id_en", value); }
        }

        /// <summary>
        /// 打印模式
        /// </summary>
		public string Print_mode {
            get { return getAttrVal<string>("Print_mode",null); }
            set { setAttrVal<string>("Print_mode", value); }
        }

        /// <summary>
        /// 页号
        /// </summary>
		public int? Page_num {
            get { return getAttrVal<int?>("Page_num",null); }
            set { setAttrVal<int?>("Page_num", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Fg_long", "Id_pat", "Id_en", "Print_mode", "Page_num"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.ordprintdto.d.OrdPrintParamDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/ordprintdto/d/OrdPrintParamDTO.java
package iih.ci.ord.dto.ordprintdto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 临床医嘱打印参数 DTO数据 
 * 
 */
public class OrdPrintParamDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 长临标识
	 * @return FBoolean
	 */
	public FBoolean getFg_long() {
		return ((FBoolean) getAttrVal("Fg_long"));
	}
	/**
	 * 长临标识
	 * @param Fg_long
	 */
	public void setFg_long(FBoolean Fg_long) {
		setAttrVal("Fg_long", Fg_long);
	}
	/**
	 * 患者Id
	 * @return String
	 */
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}
	/**
	 * 患者Id
	 * @param Id_pat
	 */
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	/**
	 * 就诊Id
	 * @return String
	 */
	public String getId_en() {
		return ((String) getAttrVal("Id_en"));
	}
	/**
	 * 就诊Id
	 * @param Id_en
	 */
	public void setId_en(String Id_en) {
		setAttrVal("Id_en", Id_en);
	}
	/**
	 * 打印模式
	 * @return String
	 */
	public String getPrint_mode() {
		return ((String) getAttrVal("Print_mode"));
	}
	/**
	 * 打印模式
	 * @param Print_mode
	 */
	public void setPrint_mode(String Print_mode) {
		setAttrVal("Print_mode", Print_mode);
	}
	/**
	 * 页号
	 * @return Integer
	 */
	public Integer getPage_num() {
		return ((Integer) getAttrVal("Page_num"));
	}
	/**
	 * 页号
	 * @param Page_num
	 */
	public void setPage_num(Integer Page_num) {
		setAttrVal("Page_num", Page_num);
	}
}